server.go

```diff
@@ -161,12 +159,26 @@ func (s *Server) Start() error {
 		return err
 	}
 
-	// Start the macaroon service and let it create its default macaroon in
-	// case it doesn't exist yet.
-	if err := s.startMacaroonService(true); err != nil {
+	// Create and start the macaroon service and let it create its default
+	// macaroon in case it doesn't exist yet.
+	s.macaroonService, err = lndclient.NewMacaroonService(
+		&lndclient.MacaroonServiceConfig{
+			DBPath:           s.cfg.BaseDir,
+			DBTimeout:        clientdb.DefaultPoolDBTimeout,
+			MacaroonLocation: poolMacaroonLocation,
+			MacaroonPath:     s.cfg.MacaroonPath,
+			Checkers: []macaroons.Checker{
+				macaroons.IPLockChecker,
+			},
+			RequiredPerms: RequiredPermissions,
+			DBPassword:    macDbDefaultPw,
```

Comment:
See my comment in `lndclient` about migrating to a non-empty password. We might need to supply the lnd client, ephemeral key (should probably export that in `lndclient` for re-use instead of only having it in LiT)